docroot/themes/humsci/su_humsci_theme/theme-settings.php

```diff
@@ -5,29 +5,8 @@
  * Provides an additional config form for theme settings.
  */
 
-use Drupal\Core\Form\FormStateInterface;
-
 /**
  * Implements hook_form_system_theme_settings_alter().
  *
  * Form override for theme settings.
  */
```

Comment:
Thanks for deleteing the function. but also delete the function comment too.

Reply:
Doh.

Reply:
Removed.